polaris-pipeline/coordinator/Domain/Dto/DurableInstanceRecord.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Newtonsoft.Json;
+
+namespace coordinator.Domain.Dto;
+
+public class DurableInstanceRecord
```

Comment:
I've been using the convention of NameDto for Dto Objects, see Common\Dto\**\*

Reply:
Ah, makes sense. Done.